python/GafferTest/HiddenFilePathFilterTest.py

```diff
@@ -45,17 +46,17 @@ class HiddenFilePathFilterTest( GafferTest.TestCase ) :
 
 	def test( self ) :
 
-		hiddenFile = Gaffer.FileSystemPath( os.path.join( self.temporaryDirectory(), ".sneaky.txt" ) )
+		hiddenFile = Gaffer.FileSystemPath( self.temporaryDirectory().joinpath( ".sneaky.txt" ).as_posix() )
```

Comment:
This is my mistake, it should be `( self.temporaryDirectory() / ".sneaky.txt" ).as_posix()` to match the pattern we're using elsewhere.

Reply:
That sounds like it would be a nice addition, and also agree that it would be fine for a future PR rather than adding to this one.